localgovdrupal/localgov_elections
# adds parent field for elections

Closes #107 

## What does this change?

Adds a "parent" field so we can use this for breadcrumbs and URL aliases.

Currently doesn't set any content types as allowed parents. If we did, it would mean a lot of dependencies for this module, where it currently has none. This means after you enable the module, you will then need to set what content types are allowed to be parents

## How to test
Enable the module, create an election (or enable election demo content module), set a node as the parent and check that the URL inherits from the parent URL.

## Have we considered potential risks?
This only applies to new installs currently, do we want an update hook to apply to current sites?
